src/ott/problems/linear/potentials.py

```diff
@@ -310,9 +313,11 @@ def plot_potential(
     X1, X2 = jnp.meshgrid(x1, x2)
     X12flat = jnp.hstack((X1.reshape(-1, 1), X2.reshape(-1, 1)))
     Zflat = jax.vmap(self.f if forward else self.g)(X12flat)
+    if displacement:
+      Zflat = 0.5 * (jnp.linalg.norm(X12flat, axis=-1) ** 2) - Zflat
     Zflat = np.asarray(Zflat)
-    vmin, vmax = np.quantile(Zflat, [quantile, 1. - quantile])
```

Comment:
Why was this removed?

Reply:
I'm sorry I forgot to clarify this. When plotting the displacement potential, the target distribution will usually be represented by the minimum of the potential. Removing the minimum 5 percent of this flattens out these minima, which in my opinion is undesirable.

Reply:
Removed it for now.